index.html

```diff
@@ -115,36 +115,36 @@ <h2>Backstory</h2>
-					As we had reached the 100 loops mark, it was obvious for us ... to do 100 more. Just to have 200 loops.<br>
-					And after we had 200 loops we liked the result very much but had no exact solution how to deal with them.
-					Bury them on just another old-fashioned linear album?<br>
-					We chose to avoid problem thinking and instead went to do what every sane person would do: make 200 more! 
+					As we reached the 100 loops mark, it was obvious for us ... to do 100 more. Just to have 200 loops.<br>
+					And after we had 200 loops, we liked the result very much but had no exact solution how to deal with them.
+					Bury them in just another old-fashioned linear album?<br>
+					We chose to avoid problematic thinking and instead went to do what every sane person would do: make 200 more! 
 					So, we started in 2008 and suddenly, 11 years later, we had 402 loops, 
 					which is because of a slight miscalculaton due to us going through the numbers backwards, but still no solution in sight
 					for an idea that came up while we were in the process of creating them. <br>
 					<br>
 			  </p>
 	 			<h2>The idea: try a new way <br>
 				<p>
-	 				So we have an endless, random album of building blocks (loops) that will each be played
-					for a random amount of time and there is limited user interaction. 
-					The effect of this is, that there is no dynamic arc that anyone could get used to.
-					There will be no imprint of the listener onto the music, since the story is always a bit different.
+	 				So now we have an endless, random album of building blocks (loops) that will each be played
+					for a random amount of time with limited user interaction. 
+					As a result, there is no dynamic arc that anyone could get used to.
+					There will be no imprint of the music onto the listener, since the story is always a bit different.
 					Therefore, this player could be either very meditative or feel like you have been visiting 
-					a funfair, with all of it's chaotic impressions.<br>
-				  Test showed, an excellent usecase is listening to it while doing (outdoor) sports.<br>
+					a fairground, with all of its chaotic impressions.<br>
+				  Testing showed that an excellent use-case is to listen to it while doing (outdoor) sports.<br>
 					Although there is quite some structure and order in this player, chaos is the main factor
-					of the overall experience. Infact it is a chaotic cyberpunk-industrial music setting. <br><br>
-					With the kind help of Rego Sen, The402 is now a neat loop-player that runs on virtually every 
-					device that has a javascript-capable browser installed.
+					of the overall experience. In fact, it is a chaotic cyberpunk-industrial music setting. <br><br>
+					With the kind help of Rego Sen's <a href="https://github.com/regosen/Gapless-5" target="_blank"><u>Gapless 5</u></a> library and additional programming, The402 is now a neat loop player that runs on virtually every 
```

Comment:
@wertstahl if you're able to incorporate this Gapless 5 link into the German version of this file, I would appreciate it!